src/netbsd.rs

```diff
@@ -56,8 +56,19 @@ fn init() -> *mut c_void {
         const POLYFILL: GetRandomFn = polyfill_using_kern_arand;
         ptr = POLYFILL as *mut c_void;
     }
-    GETRANDOM.store(ptr, Ordering::Release);
-    ptr
+    // TODO(MSRV 1.64): Consider using `Ordering::Release` instead of
+    // `Ordering::AcqRel`; see https://github.com/matklad/once_cell/issues/220
+    // and https://github.com/rust-lang/rust/pull/98383.
+    match GETRANDOM.compare_exchange(ptr::null_mut(), ptr, Ordering::AcqRel, Ordering::Acquire) {
```

Comment:
Actually, do we even need `compare_exchange` here? IIUC `dlsym` should return the same pointer on repeated calls, so I think we can use simple store with `Release` ordering.